frontend/src/app/store/migrations/migration-5.ts

```diff
@@ -0,0 +1,17 @@
+//Add personGuid in app and isReviewRequired,reviewComplete in cases
+export const Migration5 = {
```

Comment:
I would recommend not using this as a name for your migration. The fist example I crated did this as an example. You really should use a name that helps explain the migration at a glance